### PR TITLE
chore: release 13.7.0

### PR DESCRIPTION
**Pull request by cacieprins**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by jennifer-shehane — APPROVED** [review]

File: cli/CHANGELOG.md
Comment (mschile) [inline cli/CHANGELOG.md:1]: Could you also please update:

```
- Fixed a performance regression from [`13.6.4`](https://docs.cypress.io/guides/references/changelog#13-6-4) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```

to

```
- Fixed a performance regression from [`13.6.3`](https://docs.cypress.io/guides/references/changelog#13-6-3) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```

**Review by mschile — CHANGES_REQUESTED** [review]

File: cli/CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ _Released 3/12/2024 (PENDING)_
 
 **Performance:**
 
-- Fixed a performance regression from [`13.6.4`](https://docs.cypress.io/guides/references/changelog#13-6-4) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
+- Fixed a performance regression from [`13.6.3`](https://docs.cypress.io/guides/references/changelog#13-6-4) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```
Comment (mschile) [inline cli/CHANGELOG.md]:
```suggestion
- Fixed a performance regression from [`13.6.3`](https://docs.cypress.io/guides/references/changelog#13-6-3) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```

**Review by mschile — CHANGES_REQUESTED** [review]

**Review by mschile — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@cacieprins 

This PR leaves `yarn.lock` out of sync!

File: packages/example/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "cross-env": "6.0.3",
-    "cypress-example-kitchensink": "2.0.2",
+    "cypress-example-kitchensink": "2.0.3",
```
Comment (MikeMcC399) [inline packages/example/package.json:18]: `cypress-example-kitchensink` has been bumped without running `yarn`, so now `yarn.lock` is out of sync still using the previous version `"cypress-example-kitchensink": "2.0.2"`.

File: cli/CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 **Features:**
 
 - Added shadow DOM snapshot support within Test Replay in order to highlight elements correctly within the Cypress reporter. Addressed in [#28823](https://github.com/cypress-io/cypress/pull/28823).
 - Added TypeScript support for [Vue 2.7+](https://github.com/vuejs/vue/blob/main/CHANGELOG.md#270-2022-07-01). Addresses [#28591](https://github.com/cypress-io/cypress/issues/28591).
+- Adds additional context to error messages displayed when Test Replay artifacts fail to upload. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986)
 
 **Performance:**
 
-- Fixed a performance regression from [`13.6.4`](https://docs.cypress.io/guides/references/changelog#13-6-4) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
+- Fixed a performance regression from [`13.6.3`](https://docs.cypress.io/guides/references/changelog#13-6-3) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```
Comment (gabrielbahniuk) [inline cli/CHANGELOG.md:14]: @mschile regarding https://github.com/cypress-io/cypress/pull/29123#discussion_r1522052799, i'm confused, wasn't the performance regression reintroduced in 13.6.4 (and introduced the first time in 13.6.1)? From what I've tested, 13.6.1 super slow, then 13.6.3 works fine and from 13.6.4 on it doubles the time for network responses in my tests (slow again).

13.6.0 ✅ 
13.6.1 :x:
13.6.2 :x:
13.6.3 ✅ 
13.6.4 :x:
13.6.5 :x:
13.6.6 :x:

Comment (mschile) [reply cli/CHANGELOG.md:14]: @gabrielbahniuk, not that we are aware of. Can you try the pre-release binary (or wait until 13.7.0 is released) to see if it resolves your performance issue:

```
npm install https://cdn.cypress.io/beta/npm/13.7.0/linux-x64/develop-f8f7e71907870a50784a1a8547940902362b58e2/cypress.tgz
```

If your performance issue is not resolved, could you please open a new issue with a reproducible example so we can investigate.

Comment (gabrielbahniuk) [reply cli/CHANGELOG.md:14]: @mschile sure, will do and reply here, but it's weird bc I tested multiple times and everything points to a fix in 13.6.3 (that's the version I'm using right now, as 13.6.4 also didn't work).

File: packages/example/package.json (hunk 2 above)
Comment (MikeMcC399) [reply packages/example/package.json:18]: - This issue has been sorted out by https://github.com/cypress-io/cypress/pull/29128 which updated `yarn.lock`

**Pull request merged**